modules/ServerAPI/src/client/Experiment.html

```diff
@@ -90,6 +90,7 @@
         </div>
```

Comment:
Adding element for where the read-only table can be inserted into the experiment browser/editor.